metrics/cli/lib/deploy/deployer.dart

```diff
@@ -104,6 +128,36 @@ class Deployer {
     );
   }
 
+  /// Creates a new Sentry release.
+  Future<SentryConfig> _setupSentry() async {
+    final shouldSetupSentry =
+        _prompter.promptConfirm(DeployStrings.setupSentry);
+
+    if (!shouldSetupSentry) return null;
+
+    await _sentryService.login();
+
+    final release = await _createSentryRelease();
+    final dsn = _sentryService.getProjectDsn(release.project);
+
+    return SentryConfig(release: release.name, dsn: dsn);
```

Comment:
Would it be great to move this part of the code to a separate method? What do you think?